docs/Model_Assumptions.rst

```diff
@@ -2,6 +2,27 @@
 Modeling Assumptions of the MVS
 ================================
 
+Cost calculations
+-----------------
```

Comment:
Maybe you could add a link to the csvs where the costs named in this sections can be declared.

Reply:
I am not sure I know what you mean, as the costs can be declared in each of the energyAssetGroup files...

Reply:
Tried to fix it with 6cbfb1298ad7e2fdf686eb1c760cf32255f898bc

Reply:
hmm I cannot find the commit - I've searched for the first and last digits. 
A remark for future PRs: if you just post the digits that are shown on github in the PR behind each commit a link is created automatically.

Reply:
Sorry about that - I think it happened because I rebased onto dev locally, and maybe it changed all of my commit ids then? Its this one: https://github.com/rl-institut/multi-vector-simulator/pull/795/commits/52ced29aed374f0ad14841ecf2d878f795df5639